clang/unittests/Basic/FileEntryTest.cpp

```diff
@@ -179,6 +179,46 @@ TEST(FileEntryTest, DenseMapInfo) {
     EXPECT_TRUE(Set.find(R1)->isSameRef(R1));
     EXPECT_TRUE(Set.find(R2)->isSameRef(R2));
   }
+
+  // Insert R1Also second and confirm R1 "wins" when looked up as FileEntry.
```

Comment:
This comment looks like it's for the other test case.  R1Also is inserted first and it wins here.